src/state/store/configureStore.js

```diff
@@ -0,0 +1,7 @@
+import { createStore } from "redux";
+import rootReducer from "../reducers/rootReducers";
```

Comment:
You import 'rootReducer' in singular, from 'rootReducers' in plural, does this work? You export it as 'rootReducers'

Reply:
Fixed! Good spotted, but weird it still worked....

Reply:
updated now!!